main.go

```diff
@@ -17,6 +17,7 @@ var (
 	uri          = flag.String("uri", "amqp://guest:guest@localhost:5672/", "AMQP URI")
 	insecure_tls = flag.Bool("insecure-tls", false, "Insecure TLS mode: don't check certificates")
 	queue        = flag.String("queue", "", "AMQP queue name")
+        ack          = flag.Bool("ack", false, "Acknowledge messages")
```

Comment:
I think this line is indented with spaces instead of tabs which are recommended by Go style. Please use `go fmt` to format the code.